### ISSUE TITLE
[FE] 10-10d/OHI - Form Merge: Fix a11y heading issues

### ISSUE THREAD

**Issue by longmd** [department-of-veterans-affairs/va.gov-team#116800]
## Background

During implementation of the Cypress specs for the merged form, several AXE checks failed. Most of the issues are around heading levels with use of the minimal header. We need to fix these issues in order for the Cypress specs to be enabled.

## Scope
- Identify all a11y defects in the form
- Fix all issues as outlined in the AXE report

## Acceptance Criteria/Definition of Done
-  App is free from a11y defects
